src/main/scala/com/cognite/sdk/scala/v1/dataModels.scala

```diff
@@ -10,6 +10,11 @@ final case class DataModelIdentifier(
     model: String
 )
 
+final case class DataModelInstanceIdentifier(
```

Comment:
Since we abbreviate it to "Id" in both "externalId" and [internal] "id" it makes sense to shorten it to "Id" as well, to be consistent.